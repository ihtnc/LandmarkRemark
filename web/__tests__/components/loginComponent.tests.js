import React from 'react';
import { shallow } from 'enzyme';
import Login from "@components/loginComponent";

describe("LoginComponent", () => {
  test("should match latest snapshot", () => {
    const wrapper = shallow(<Login />)
    expect(wrapper).toMatchSnapshot();
  });
});