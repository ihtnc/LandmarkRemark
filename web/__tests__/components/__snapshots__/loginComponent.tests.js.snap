// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginComponent should match latest snapshot 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      Email:
    </styled.div>
    <styled.input
      onChange={[Function]}
      value=""
    />
  </styled.div>
  <styled.div>
    <styled.div>
      Password:
    </styled.div>
    <styled.input
      onChange={[Function]}
      type="password"
      value=""
    />
  </styled.div>
  <styled.span
    error={false}
    show={false}
  />
  <styled.div>
    <styled.button
      disabled={false}
      onClick={[Function]}
    >
      Login
    </styled.button>
    <styled.button
      disabled={false}
      onClick={[Function]}
    >
      Register
    </styled.button>
  </styled.div>
</styled.div>
`;
